### ISSUE TITLE
Have unique IfcGloballyUniqueId within the model suite

### ISSUE THREAD

**Issue by thbeu** [buildingSMART/IFC4.3.x-sample-models#20]
The set IFC object identifiers are not unique within the model suite (since the files are likely manually crafted). It would be preferable if these identifers are unique such that importing applications can handle more than one IFC file simultaneously.

**Comment by Andrej730** [comment]
Sounds like it's an issue with the importing application that it cannot handle overlapping guids from loaded models - since it is possible that user would load multiple versions of the same model e.g. to investigate the changes.

**Comment by thbeu** [comment]
> Sounds like it's an issue with the importing application that it cannot handle overlapping guids from loaded models - since it is possible that user would load multiple versions of the same model e.g. to investigate the changes.

Yes, it's likely that different version of the same model might lead to importer issues. However, for the sample repository we do not have a version handling as different IFC sample represent different models.
